web/app/js/components/util/Utils.js

```diff
@@ -55,7 +55,7 @@ export const metricToFormatter = {
   "SUCCESS_RATE": m => _isNil(m) ? "---" : successRateFormatter(m),
   "LATENCY": formatLatencyMs,
   "UNTRUNCATED": m => styleNum(m, "", false),
-  "BYTES": m => _isNil(m) ? "---" : styleNum(m, "B", true),
+  "BYTES": m => _isNil(m) ? "---" : styleNum(m, "bps", true),
```

Comment:
naming nit: `bps` is typically bits per second. consider `B/s` 
https://en.wikipedia.org/wiki/Data-rate_units

Reply:
oooh thanks!

Reply:
changing this to B/s as it seems more standard
cc @grampelberg